emc-metalnx-services/src/main/java/com/emc/metalnx/services/configuration/ConfigServiceImpl.java

```diff
@@ -86,6 +86,9 @@ public class ConfigServiceImpl implements ConfigService {
 
 	@Value("${pluggableshoppingcart.enabled}")
 	private boolean pluggableShoppingCartEnabled = false;
+	
+	@Value("${sidebar.show.public:false}")
+	private boolean publicSidebarLinkEnabled;
```

Comment:
Is this about showing a link to the "public" group?

I take it the annotation is linking configuration to the variable and also providing a default?

Reply:
Yes, this links the `sidebar.show.public` configuration setting (in metalnx.properties) to the `publicSidebarLinkEnabled` variable.  And, yes, this provides the default of `false`.